FishyTransport/Sources/FishyActorsCore/Analysis.swift

```diff
@@ -205,9 +209,9 @@ extension FunctionSignatureSyntax {
 // ==== ----------------------------------------------------------------------------------------------------------------
 // MARK: Analysis decls
 
-struct DistributedActorDecl {
+public struct DistributedActorDecl {
   let access: AccessControl
-  let name: String
+  public let name: String
   var funcs: [FuncDecl]
 }
```

Comment:
this was an artefact of wanting to mess as little with the project as possible during the restructuring, but since it is no longer needed I just removed public everywhere.